src/BibleWell.App/ViewModels/MainViewModel.cs

```diff
@@ -35,15 +59,48 @@ public MainViewModel(Router router)
         }
     }
 
-    public static ObservableCollection<MenuItemTemplate> MenuItems { get; } =
-    [
-        new(typeof(HomePageViewModel), "HomeRegular"),
-        new(typeof(BiblePageViewModel), "BookOpenRegular"),
-        new(typeof(GuidePageViewModel), "CompassNorthwestRegular"),
-        new(typeof(ResourcesPageViewModel), "ClipboardRegular"),
-        new(typeof(LibraryPageViewModel), "LibraryRegular"),
-        new(typeof(DevPageViewModel), "WindowDevToolsRegular"),
-    ];
+    private void RegisterExperienceChangedHandler()
+    {
+        WeakReferenceMessenger.Default.Register<ExperienceChangedMessage>(
+            this,
+            (_, m) =>
+            {
+                Experience = m.Value;
+                _userPreferencesService.Set(PreferenceKeys.Experience, (int)m.Value);
+                InitializeMenuItems();
+            });
+    }
+
+    private void InitializeMenuItems()
+    {
+        MenuItems.Clear();
+        switch (Experience)
+        {
+            case AppExperience.None:
+                NavMenuVisible = false;
+                break;
+            case AppExperience.Default:
+                NavMenuVisible = true;
+                MenuItems.Add(new MenuItemTemplate(typeof(HomePageViewModel), "WellIcon", true, "WellIconInactive"));
+                MenuItems.Add(new MenuItemTemplate(typeof(BiblePageViewModel), "BookOpenRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(GuidePageViewModel), "CompassNorthwestRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(ResourcesPageViewModel), "ClipboardRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(LibraryPageViewModel), "LibraryRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(DevPageViewModel), "WindowDevToolsRegular"));
+                break;
+            case AppExperience.Fia:
+                NavMenuVisible = true;
+                MenuItems.Add(new MenuItemTemplate(typeof(HomePageViewModel), "WellIcon", true, "WellIconInactive"));
+                MenuItems.Add(new MenuItemTemplate(typeof(BiblePageViewModel), "BookOpenRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(GuidePageViewModel), "CompassNorthwestRegular"));
+                MenuItems.Add(new MenuItemTemplate(typeof(ResourcesPageViewModel), "ClipboardRegular"));
+                break;
+            // disable below because there are compiler errors when it goes out
+            // ReSharper disable once RedundantEmptySwitchSection
+            default:
+                break;
```

Comment:
This edge case should probably log an error and then send the user to the welcome page screen to pick an experience.